client/components/checklist/task.js

```diff
@@ -178,45 +187,45 @@ class Task extends PureComponent {
 			>
 				<div className="checklist__task-primary">
 					<h3 className="checklist__task-title">
-						{ completed ? (
-							completedTitle || title
-						) : (
+						{ hasActionlink ? (
 							<Button
 								borderless
 								className="checklist__task-title-link"
 								href={ href }
-								onClick={ this.onTaskClick }
+								onClick={ this.onTaskTitleClick }
 								target={ target }
 							>
-								{ title }
+								{ completed ? completedTitle : title }
+								{ this.renderChevron() }
```

Comment:
This can be simplified:

```suggestion
								<Gridicon icon={ this.isTaskCollapsed() ? 'chevron-down' : 'chevron-up' } />;
```

...& we can remove the `renderChevron` function if we don't plan on expanding it straightaway.

Reply:
That's much cleaner, thanks for the suggestion.

Reply:
Ah, then ignore my comment https://github.com/Automattic/wp-calypso/pull/33666/files#r291427767 :)